test/test-schema.js

```diff
@@ -1,23 +1,24 @@
 'use strict';
 const Ajv = require('ajv');
+const betterAjvErrors = require('better-ajv-errors');
 const path = require('path');
-const ajv = new Ajv({ allErrors: true });
+
+const ajv = new Ajv({ jsonPointers: true, allErrors: true });
```

Comment:
Why was this added? `jsonPointers: true,` ?

Reply:
Because `better‑ajv‑errors` requires it&nbsp;to&nbsp;function.

Reply:
Ah okay, cool! :+1: